### PR TITLE
Changes for April 2024

### PR DESCRIPTION
**Pull request by adderall-prozac**
Notes:

- These changes are being rolled out during the week, so you may not see the changes right away.
- The 2:0 schema can be used by the APIs approximately a week from this release date. We are approximating APIs can use this schema on April 8.

### CODE REVIEW

**Review by aisling-2 — APPROVED** [review]

**Comment by octavio12345300** [comment]
Gracias

**Comment by chris6090** [comment]
Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Review by octavio12345300 — COMMENTED** [review]
Hola

File: sbe/schemas/spot_latest.xml
Comment (Alkhtri77) [inline sbe/schemas/spot_latest.xml:1]: Hello help me

**Review by Alkhtri77 — COMMENTED** [review]
Hello help me plaes

File: sbe/schemas/spot_latest.xml
Comment (chris6090) [reply sbe/schemas/spot_latest.xml:1]: Chris Nwachukwu

Comment (chris6090) [reply sbe/schemas/spot_latest.xml:1]: > Hello help me 

Chris Nwachukwu

**Comment by chris6090** [comment]
> Approve merging please 

Chris Nwachukwu 
Kriseng83@gmail.com

**Comment by chris6090** [comment]
> Notes:
> 
> 
> 
> - These changes are being rolled out during the week, so you may not see the changes right away.
> 
> - The 2:0 schema can be used by the APIs approximately a week from this release date. We are approximating APIs can use this schema on April 8.
> 
> 

Chris Nwachukwu 
Kriseng83@gmail.com

File: sbe/schemas/spot_latest.xml
Comment (octavio12345300) [reply sbe/schemas/spot_latest.xml:1]: 👍

**Comment by chris6090** [comment]
________________________________

**Pull request merged**